curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/networking-write-a-web-browser.english.md

```diff
@@ -37,10 +37,14 @@ question:
     ```
```

Comment:
It's slightly out of scope of this PR, but the indentation is wrong above.  Could you line everything (lines 33 through 34) up with `data`?

It should be

```py
while True:
    data = mysock.recv(512)
    if len(data) < 1:
        break
    print(data.decode(),end='')
mysock.close()
```